python-checks/src/test/java/org/sonar/python/checks/UseOfAnyAsTypeHintCheckTest.java

```diff
@@ -42,4 +43,15 @@ void useOfUserDefinedTypeCalledAny() {
   void useOfOverrideOrOverloadDecorator() {
     PythonCheckVerifier.verify(List.of("src/test/resources/checks/useOfOverrideOrOverloadDecorator.py", "src/test/resources/checks/reexport_typing_overload_override.py"), new UseOfAnyAsTypeHintCheck());
   }
+
+  @Test
+  void useOfAnyImpored() {
+    PythonCheckVerifier.verify(
+      List.of(
+        "src/test/resources/checks/useOfAnyAsTypeHintImported.py",
+        "src/test/resources/checks/useOfAnyAsTypeHintImporting.py"
```

Comment:
__minor__ Since there are many existing files used to test this rule, I suggest regrouping them in a folder, for example in `src/test/resources/checks/useOfAnyAsTypeHint`